tastypie/authentication.py

```diff
@@ -8,7 +8,15 @@
 from django.conf import settings
 from django.contrib.auth import authenticate
 from django.core.exceptions import ImproperlyConfigured
-from django.middleware.csrf import _sanitize_token
+
```

Comment:
We try to put this sort of compatibility stuff into `compat.py` to avoid cluttering the other modules with too much of it.  It also provides opportunity to document why, and re-use the same fixes in multiple places.  The token sanitization stuff is a recurring issue so there's already code in `compat.py` to work with it.